agent/api/appmesh/appmesh_test.go

```diff
@@ -72,8 +97,8 @@ func TestAppMeshFromACSNonAppMeshProxyInput(t *testing.T) {
 func prepareProxyConfig() ecsacs.ProxyConfiguration {
 
 	return ecsacs.ProxyConfiguration{
-		Type:          aws.String(appMesh),
-		Properties:    map[string]*string{
+		Type: aws.String(appMesh),
```

Comment:
Why the space is deleted? Can you run go fmt?

Reply:
my go fmt did this (removed the space)

Reply:
i guess you two have different versions of gofmt